process/block/preprocess/gasConsumption_test.go

```diff
@@ -337,7 +338,7 @@ func TestComputeGasConsumedByMiniBlock_ShouldWork(t *testing.T) {
 				return 6
 			},
 		},
-		&mock.TxTypeHandlerMock{ComputeTransactionTypeCalled: func(tx data.TransactionHandler) (process.TransactionType, process.TransactionType) {
+		&testscommon.TxTypeHandlerMock{ComputeTransactionTypeCalled: func(tx data.TransactionHandler) (process.TransactionType, process.TransactionType) {
```

Comment:
you could move `ComputeTransactionTypeCalled` on a new line (also in the entire file)

Reply:
fixed